packages/create-vite/template-preact-ts/tsconfig.json

```diff
@@ -5,7 +5,12 @@
     "module": "ESNext",
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "skipLibCheck": true,
-
+    "baseUrl": "./",
```

Comment:
Is this `baseUrl` needed? The downside with this is that TypeScript would autocomplete imports as `"src/components/..."` (note leading `src/`), which wouldn't work in Vite ootb.

Reply:
Ah interesting, I didn't know that thanks.  I'm using the `baseUrl` in this vitejs project: https://github.com/AJamesPhillips/conversation-helper and when I use VisualStudio code to add a missing import it adds as a relative path like ["./components/LogList"](https://github.com/AJamesPhillips/conversation-helper/blob/main/src/app.tsx#L4C3-L4C3).  How do you recreate the autocomplete imports as "src/..." is this a setting I have set that I'm now unaware of perhaps?

Reply:
It usually happens when you have a very long path of `../../../../../../` and TS intelligently picks the short `src/components/..` path instead. But anyways, setting this `baseUrl` also let TS resolves correctly without warnings, so a user who manually type that (for any reason) would falsely think it works too.

Reply:
Ok.  Makes sense.  I just wish there was a more descriptive error message to point you towards adding "baseUrl" and "paths" to tsconfig.json

Sounds like you don't want this contribution so I'll close this PR.

Reply:
I do want the `paths` contribution! Just not the `baseUrl` part. Is the `baseUrl` required for `paths` to work?

Reply:
Ah ok.  I misunderstood.  In my own project it seems I can remove "baseUrl" and imports, type checking, and vite dev/build all work without error so I will hesitantly say yes but I don't really know why [preact would recommend adding baseUrl](https://preactjs.com/guide/v10/typescript/#typescript-preactcompat-configuration) if it wasn't needed.